pkg/controller/persist/persist.go

```diff
@@ -0,0 +1,6 @@
+package persist
```

Comment:
These values look to only be used in the cluster package. They don't need their own package and should be moved to cluster.go unexported.